DQM/Integration/python/config/inputsource_cfi.py

```diff
@@ -67,7 +67,7 @@
         SelectEvents = cms.untracked.vstring('*'),
         streamLabel = cms.untracked.string('streamDQM'),
         scanOnce = cms.untracked.bool(options.scanOnce),
-        minEventsPerLumi = cms.untracked.int32(1),
+        minEventsPerLumi = cms.untracked.int32(-1),
```

Comment:
@cobby319 this is a general DQM configuration file for Online, this is not supposed to be changed for subsystems use. Please revert back this change.